bciers/apps/registration/app/components/transfers/fetchTransferEventsPageData.ts

```diff
@@ -15,7 +15,9 @@ export const formatTransferRows = (rows: GridRowsProp) => {
       created_at,
     }) => {
       return {
-        id,
+        // We need to generate a unique id for each row to avoid issues with the DataGrid(MUI requires a unique id for each row)
+        id: Math.random(), // NOSONAR
+        transfer_id: id,
```

Comment:
This looks fine, but we could also use the `getRowId` prop or make an id out of the id + the facility name or something else unique

Reply:
Yes, when I was implementing this, `getRowId` hadn't been added to the DataGrid yet, and I wanted to avoid creating a conflict by adding it myself. However, now that it's available, I can use it.